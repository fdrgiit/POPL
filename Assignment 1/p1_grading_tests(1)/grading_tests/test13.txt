
`'"?|\

(j.1INTx.2REAL)

ASSIGN.1INt

