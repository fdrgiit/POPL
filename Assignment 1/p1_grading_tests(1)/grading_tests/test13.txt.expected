2 38 ERROR
2 38 ERROR
2 38 ERROR
2 38 ERROR
2 38 ERROR
2 38 ERROR
4 24 (
4 35 j.1INT 
4 36 x.2REAL 
4 25 )
6 4 ASSIGN
6 33 .
6 34 1 
6 37 INt 
